run_dir/static/js/running_notes.js

```diff
@@ -100,13 +100,25 @@ function load_running_notes(wait) {
   // Clear previously loaded notes, if so
   note_url = get_note_url();
   $("#running_notes_panels").empty();
+  // From agreements tab
+  $("#invoicing_notes").empty();
+  inv_notes = [];
   return $.getJSON(note_url, function(data) {
     if(Object.keys(data).length == 0 || typeof data === 'undefined'){
       $('#running_notes_panels').html('<div class="well">No running notes found.</div>');
     } else {
       $.each(data, function(date, note) {
         $('#running_notes_panels').append(make_running_note(date, note));
+        if(note['category'].includes('Invoicing')){
+          inv_notes.push(date, note);
```

Comment:
Make it a single object. Now its just pushing 2 elements into the list.

Reply:
Good idea!